cmd/expiration-mailer/main.go

```diff
@@ -292,96 +293,124 @@ func (ds durationSlice) Swap(a, b int) {
 
 const clientName = "ExpirationMailer"
 
+type config struct {
+	Mailer struct {
+		cmd.ServiceConfig
+		cmd.DBConfig
+		cmd.SMTPConfig
+
+		From    string
+		Subject string
+
+		CertLimit int
+		NagTimes  []string
+		// How much earlier (than configured nag intervals) to
+		// send reminders, to account for the expected delay
+		// before the next expiration-mailer invocation.
+		NagCheckInterval string
+		// Path to a text/template email template
+		EmailTemplate string
+	}
+
+	Statsd cmd.StatsdConfig
+
+	Syslog cmd.SyslogConfig
+}
+
 func main() {
-	app := cmd.NewAppShell("expiration-mailer", "Sends certificate expiration emails")
-
-	app.App.Flags = append(app.App.Flags, cli.IntFlag{
-		Name:   "cert_limit",
-		Value:  100,
-		EnvVar: "CERT_LIMIT",
```

Comment:
@benileo Did any of your other refactors remove codegansta CLI flags with an `EnvVar` configured? This is the first one I can remember noticing. 

I want to make sure that we aren't setting `CERT_LIMIT` in staging/prod since it will stop working after this PR. @jsha - do you know off hand? I'm guessing we favor explicit CLI arguments over ENV vars but it merits checking.